tools/templates/L10N/HU

```diff
@@ -0,0 +1,752 @@
+# This file contains that German translation of features of the
```

Comment:
s/that German/the Hungarian/ (assuming of course that “German” is not Hungarian for “Hungarian”)

Reply:
Good catch... most definitely work in progress.